packages/ui/client/components/views/ViewModuleGraph.vue

```diff
@@ -6,9 +6,23 @@ import type { ModuleGraph, ModuleGraphController, ModuleLink, ModuleNode, Module
 
 const props = defineProps<{
   graph: ModuleGraph
+  headerSize: number
 }>()
 
-const { graph } = toRefs(props)
+const header = ref(null)
+const thisHeaderSize = ref<number>(0)
+
+const { graph, headerSize } = toRefs(props)
+
+useResizeObserver(header, () => {
+  const clientHeight = unrefElement(header)?.clientHeight
+  thisHeaderSize.value = clientHeight ?? 0
+})
+
+const style = computed(() => {
+  const size = headerSize.value + thisHeaderSize.value
+  return size > 0 ? `--graph-height: calc(100vh - ${size}px)` : null

```

Comment:
Maybe we could use [<style> v-bind](https://blog.vuejs.org/posts/vue-3.2.html) instead of the computed styles, but this works well anyway.

Reply:
I think we don't need include a style + a `v-bind` to do this